config/samples/apps_v1alpha1_nemoguardrails.yaml

```diff
@@ -3,61 +3,57 @@ kind: NemoGuardrails
 metadata:
   labels:
     app.kubernetes.io/name: k8s-nim-operator
-    app.kubernetes.io/managed-by: kustomize
+    app.kubernetes.io/managed-by: k8s-nim-operator
   name: nemoguardrails-sample
 spec:
   authSecret: ngc-api-secret
   configStore:
-	configMap: gr-config
+    configMap: gr-config
   env:
-  - name: NEMO_GR_SVC_SERVICE_PORT
-	value: "7331"
-  - name: NEMO_GR_SVC_SERVICE_PORT
-	value: "7331"
-  - name: NIM_ENDPOINT_URL
-	value: http://<NIM_SERVICE_ENDPOINT>:<NIM_SERVICE_PORT>/v1
-  - name: NIM_ENDPOINT_API_KEY
-	value: dummy
+    - name: NIM_ENDPOINT_URL
+      value: "http://<NIM_SERVICE_ENDPOINT>:<NIM_SERVICE_PORT>/v1"
+    - name: NIM_ENDPOINT_API_KEY
+      value: dummy
   expose:
-	ingress:
-  	spec: {}
-	service:
-  	port: 7331
-  	type: ClusterIP
+    ingress:
+      spec: {}
+    service:
+      port: 7331
+      type: ClusterIP
   image:
-	pullPolicy: IfNotPresent
-	pullSecrets:
-	- ngc-secret
-	repository: nvcr.io/nvidian/nemo-llm/nemo-guardrails-microservice
-	tag: "25.02-rc4"
+    pullPolicy: IfNotPresent
+    pullSecrets:
+      - ngc-secret
+    repository: nvcr.io/nvidian/nemo-llm/nemo-guardrails-microservice
+    tag: "25.02-rc4"
```

Comment:
can this be changed to 25.03-rcxx

Reply:
thanks @mkhaas updating it as part of other MR.